src/components/molecules/LogOutput/LogOutputActions.tsx

```diff
@@ -16,9 +19,15 @@ type LogOutputActionsProps = {
 
 const LogOutputActions: React.FC<LogOutputActionsProps> = props => {
   const {logOutput, actions} = props;
+  const isSecureContext = useSecureContext();
 
   const logOutputActionsMap: {[key in LogAction]: any} = {
-    copy: <CopyButton content={stripAnsi(logOutput)} key="copy-action" />,
+    copy: isSecureContext ? (
+      <CopyButton content={stripAnsi(logOutput)} key="copy-action" />
+    ) : (
+      // TODO: (Punksage): Find out the name to save the file
```

Comment:
should it be dynamic name or based on execution id?

Reply:
@vsukhin I can be either way. How about the format {execution-name}-{hash-of-the-content}.log?